index.css

```diff
@@ -1,75 +1,138 @@
+*{padding:0;margin:0;border:0;}
+*,*:before,*:after{-moz-box-sizing:border-box;-webkit-box-sizing:border-box;box-sizing:border-box;}
+:focus,:active{outline:none;}
+a:focus,a:active{outline:none;}
+nav,footer,header,aside{display:block;}
+html,body{height:100%;width:100%;font-size:100%;line-height:1;font-size:14px;-ms-text-size-adjust:100%;-moz-text-size-adjust:100%;-webkit-text-size-adjust:100%;overflow-x: hidden;}
+input,button,textarea{font-family:inherit;}
+input::-ms-clear{display:none;}
+button{cursor:pointer;}
+button::-moz-focus-inner{padding:0;border:0;}
+a,a:visited{text-decoration:none;}
+a:hover{text-decoration:none;}
+ul li{list-style:none;}
+img{vertical-align:top;}
+h1,h2,h3,h4,h5,h6{font-size:inherit;font-weight:inherit;}
+
+/* ------------------------------------------------------------------------------------ */
+html, body{
+  height: 100%;
 }
 
-*, *:before, *:after {
-  -webkit-box-sizing: border-box;
-  box-sizing: border-box;
+body{
+ min-height: 100%;
+ display: flex;
+ flex-direction: column;
 }
 
-:focus, :active {
-  outline: none;
+.wrapper {
+  margin: 20px;
 }
 
-a:focus, a:active {
-  outline: none;
+.container {
+  max-width: 1300px;
+  margin: 0 auto;
 }
 
-nav, footer, header, aside {
-  display: block;
+header{
+  background-color: aquamarine;
 }
 
-html, body {
-  height: 100%;
-  width: 100%;
-  font-size: 100%;
-  line-height: 1;
-  font-size: 14px;
-  -ms-text-size-adjust: 100%;
-  -moz-text-size-adjust: 100%;
-  -webkit-text-size-adjust: 100%;
+.header-flex {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header-logo a{
+  font-size: 40px;
+  text-transform: uppercase;
+  color: #000;
+}
+
+.header-navigation ul{
+  display: flex;
 }
 
-input, button, textarea {
-  font-family: inherit;
+.header-navigation li a{
+  margin: 0 20px 0 0;
+  color: black;
 }
 
-input::-ms-clear {
-  display: none;
+.header-info ul{
+  display: flex;
 }
 
-button {
-  cursor: pointer;
+.header-info li a{
+  margin: 0 20px 0 0;
+  color: black;
 }
 
-button::-moz-focus-inner {
-  padding: 0;
-  border: 0;
+.header-info li a:hover, .header-navigation li a:hover{
+  text-decoration: underline;
 }
+/* ------------------------------------------------------------- */
 
-a, a:visited {
-  text-decoration: none;
+main{
+  min-height: calc(100vh - 80px);
 }
 
-a:hover {
-  text-decoration: none;
+.top-baner-block {
+  background: url(https://manrule.ru/images/article/orig/2020/03/muzhskaya-odezhda-lacoste-assortiment-i-tablica-razmerov.jpg) no-repeat;
+  height: 750px;
 }
 
-ul li {
-  list-style: none;
+.top-baner-block-text {
+  max-width: 1300px;
+  margin: 200px auto;
+  z-index: 2;
 }
 
-img {
-  vertical-align: top;
+.top-baner-item {
+  font-size: 42px;
 }
 
-h1, h2, h3, h4, h5, h6 {
-  font-size: inherit;
-  font-weight: inherit;
+.top-baner-text {
+  font-size: 28px;
+  margin: 10px 0 ;
+}
+
+.top-baner-link {
+  display: inline-block;
+  padding: 10px 20px;
+  background-color: seagreen;
```

Comment:
It's better to use #hex format of colors in styles, for example: #ffffff - white, #000000 - black, #2E8B57 - seagreen